aip/client-libraries/4221.md

```diff
@@ -68,53 +68,75 @@ For example, gRPC supports the [resolution of client configuration][1] that
 
 Client libraries **should** respect the remotely-resolved configuration, except
-when a user overrides it via the aforementioned client library [retry
-configuration mechanisms](#retry-configuration-mechanisms).
+when a user overrides it via the aforementioned client library
+[retry configuration mechanisms](#retry-configuration-mechanisms).
 
 ### Client library generator retry configuration
 
 Client library generators that implement client-side retry **must** accept a
 retry configuration. This is to enable API producers to supply a retry
 configuration that best-suits their service.
 
-For example, gRPC client-side retry
-is configured with a [RetryPolicy][3] within a [gRPC Service Config][4]. Here is
-a configuration that applies the `RetryPolicy` to all methods in the
-`google.example.library.v1.LibraryService` service, except for those that are
-explicitly named which get no `RetryPolicy`.
+For example, gRPC client-side retry is configured with a [RetryPolicy][3]
+within a [gRPC Service Config][4]. Here is a configuration that applies the
+`RetryPolicy` to all methods in the `google.example.library.v1.LibraryService`
+service, except for those that are explicitly named which get no `RetryPolicy`.
 
 ```json
 {
-  "methodConfig": [{
-    "name": [{ "service": "google.example.library.v1.LibraryService" }],
-    "waitForReady": true,
-    "timeout": "60s",
-    "retryPolicy": {
-      "maxAttempts": 3,
-      "initialBackoff": "0.01s",
-      "maxBackoff": "60s",
-      "backoffMultiplier": 1.3,
-      "retryableStatusCodes": ["UNAVAILABLE", "UNKNOWN"]
+  "methodConfig": [
+    {
+      "name": [{ "service": "google.example.library.v1.LibraryService" }],
+      "waitForReady": true,
+      "timeout": "60s",
+      "retryPolicy": {
+        "maxAttempts": 3,
+        "initialBackoff": "0.01s",
+        "maxBackoff": "60s",
+        "backoffMultiplier": 1.3,
+        "retryableStatusCodes": ["UNAVAILABLE", "UNKNOWN"]
+      }
+    },
+    {
+      "name": [
+        {
+          "service": "google.example.library.v1.LibraryService",
+          "method": "CreateShelf"
+        },
+        {
+          "service": "google.example.library.v1.LibraryService",
+          "method": "DeleteShelf"
+        },
+        {
+          "service": "google.example.library.v1.LibraryService",
+          "method": "MergeShelves"
+        },
+        {
+          "service": "google.example.library.v1.LibraryService",
+          "method": "CreateBook"
+        },
+        {
+          "service": "google.example.library.v1.LibraryService",
+          "method": "DeleteBook"
+        },
+        {
+          "service": "google.example.library.v1.LibraryService",
+          "method": "UpdateBook"
+        },
+        {
+          "service": "google.example.library.v1.LibraryService",
+          "method": "MoveBook"
+        }
+      ],
+      "waitForReady": true,
+      "timeout": "60s"
     }
-  },
-  {
-    "name": [
```

Comment:
Can we avoid formatting JSON? I prefer the original compact configuration here.

Reply:
I do too, and there is an ignore mechanic. I will add it here.

Reply:
Done.